scripts/copy_sample_cram_to_release.py

```diff
@@ -40,24 +40,33 @@ def check_paths_exist(paths: list[str]):
     return True
 
 
-def copy_to_release(project: str, paths: list[str]):
+def copy_to_release(project: str, billing_project: str, paths: list[str]):
     """
     Copy many files from main bucket paths to the release bucket with todays date as directory
     """
     today = time.strftime('%Y-%m-%d')
     release_path = f'gs://cpg-{project}-release/{today}/'
 
     subprocess.run(
```

Comment:
Not specifically related to this change, but can we wrap this in a try-except and report the command line error before the job dies to make this type of debugging easier in future?